README.md

```diff
@@ -1,2 +1,33 @@
-# loopback-sdk-ios-codegen
-Code generation tool for LoopBack iOS SDK
+# LoopBack iOS SDK CodeGen
+
+The iOS CodeGen command line tool generates iOS client side LoopBack Model representations
+in Objective-C by looking into the specified server application.
+
+## How to Use
+
+The following is an example of the usage applied to the test server provided under this directory:
+
+ 1. Run `npm install` in this directory to initialize the tool `bin/lb-ios` and the test server under `test-env/server`.
+ *  Run `bin/lb-ios -p XX test-env/server/server test-env/client/ios/gen-src` 
+ 	to generate the models in Objective-C representation, 
+ 	where `XX` is the prefix attached to all the generated class names.
+
+## How to Run the Unit Test
+
+After performing the above steps, run the followings:
+
+ 1. Run the test server by `node test-env/server/server.js &`.
+ *  Copy `LoopBack.framework` into `test-env/CodeGenTest/`.
```

Comment:
Sounds good!  I guess I'll ping @raymondfeng when I finish the first cut of method codegen as I'm finding (minor) improvements in the iOS SDK code as I work on the method support (I'm accumulating the fixes in this PR: https://github.com/strongloop/loopback-sdk-ios/pull/67)  I may also work on Buffer and GeoPoint types support before asking the release (I've added Date type support but not yet worked on those two).